testcontainers/tests/cli_client.rs

```diff
@@ -22,20 +23,16 @@ impl Image for HelloWorld {
 #[tokio::test(flavor = "multi_thread")]
 async fn cli_can_run_hello_world() {
     let _ = pretty_env_logger::try_init();
-
-    let docker = clients::Cli::default();
-
-    let _container = docker.run(HelloWorld);
+    let _container = HelloWorld.start();
 }
 
 #[test]
 fn generic_image_with_custom_entrypoint() {
-    let docker = clients::Cli::default();
     let msg = WaitFor::message_on_stdout("server is ready");
 
     let generic = GenericImage::new("simple_web_server", "latest").with_wait_for(msg.clone());
 
-    let node = docker.run(generic);
+    let node = generic.start();
```

Comment:
What I like about this new API is that we can chain it, can we rewrite the tests to do that?